Libraries/Core/ReactFiberErrorDialog.js

```diff
@@ -24,7 +25,7 @@ import {handleException, SyntheticError} from './ExceptionsManager';
  * trace within the native redbox component.
  */
 function showErrorDialog(capturedError: CapturedError): boolean {
-  const {componentStack, error} = capturedError;
+  const {componentStack, error, errorBoundaryFound} = capturedError;
```

Comment:
Who sets the `errorBoundaryFound` property? I cannot seem to find it.

Reply:
I have no idea, but I'll try to check. I thought React does it. I was testing it under React `16.14.0`.

Reply:
I can only find a `errorBoundary` property: https://github.com/facebook/react-native/blob/c7ab78036070b70c014c15029504caa43eaf6925/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js#L5220

Maybe `errorBoundaryFound` was in a previous version.

Reply:
The type passed into `showErrorDialog` injection has been changed in React `17.0.0` 

https://github.com/facebook/react/blob/17.0.1/packages/react-reconciler/src/ReactFiberErrorLogger.js#L22

Now it's a `Fiber` type 

https://github.com/facebook/react/blob/master/packages/react-reconciler/src/ReactInternalTypes.js#L65-L192

But in React `16.0.0` it was a `CapturedError` type with a lot of fields related to error boundary

https://github.com/facebook/react/blob/16.8.6/packages/react-reconciler/src/ReactFiberErrorLogger.js#L14

AFAIK React Native don't plan to support React v17 for a while. When it will, I think we can change logic for resolving error catching by boundary. Nothing will break after switching to React v17, but the functionality described in this pr will stop working.

Potentially I could add more logic for React v17 case, but I am pretty sure I can't even start RN app on app (was not able to do it few weeks ago).

Reply:
> I can only find a errorBoundary property:

Ok, I got what you meant. I am not familiar with RN source code and didn't know that there is a different Renderer. I'll change the logic to relay just on `errorBoundary` property and not on `errorBoundaryFound`.

Reply:
Done. Thx for the help!

Reply:
Also, I've tested everything again and it works as expected.